mne/preprocessing/ica.py

```diff
@@ -1370,6 +1371,63 @@ def plot_components(self, picks=None, ch_type=None, res=64, layout=None,
                                    image_interp=image_interp,
                                    head_pos=head_pos)
 
+    def plot_properties(self, inst, picks=None, axes=None, dB=True,
+                        cmap=None, plot_std=True, topo_args=None,
+                        image_args=None, psd_args=None, figsize=None,
+                        show=True):
+        """Display component properties: topography, epochs image, ERP,
+        power spectrum and epoch variance.
+
+        Parameters
+        ----------
+        inst: instance of Epochs or Raw
+            The data to use in plotting properties.
+        picks : int | array-like of int | None.
+            The components to be displayed. If None, plot will show the first
+            five sources in the order as fitted. If more than one components
```

Comment:
copied from another docstring, but I agree.Probably just saying `... plot will show the first five sources.` would be better. just BTW - mne ICA does not sort by variance explained, right?